WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/compose/DomainItem.kt

```diff
@@ -2,57 +2,151 @@ package org.wordpress.android.ui.sitecreation.domains.compose
-import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
 import androidx.compose.material.Divider
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.wordpress.android.ui.compose.components.SolidCircle
+import org.wordpress.android.ui.compose.theme.AppColor
 import org.wordpress.android.ui.compose.theme.AppTheme
+import org.wordpress.android.ui.compose.unit.Margin
+import org.wordpress.android.ui.compose.utils.asString
 import org.wordpress.android.ui.sitecreation.domains.SiteCreationDomainsViewModel.ListItemUiState.New.DomainUiState
+import org.wordpress.android.ui.sitecreation.domains.SiteCreationDomainsViewModel.ListItemUiState.New.DomainUiState.Cost
+import org.wordpress.android.ui.sitecreation.domains.SiteCreationDomainsViewModel.ListItemUiState.New.DomainUiState.Variant.BestAlternative
+import org.wordpress.android.ui.sitecreation.domains.SiteCreationDomainsViewModel.ListItemUiState.New.DomainUiState.Variant.Recommended
+import org.wordpress.android.ui.sitecreation.domains.SiteCreationDomainsViewModel.ListItemUiState.New.DomainUiState.Variant.Sale
+import org.wordpress.android.ui.sitecreation.domains.SiteCreationDomainsViewModel.ListItemUiState.New.DomainUiState.Variant.Unavailable
 
-private val CostColorDefault @Composable get() = MaterialTheme.colors.onSurface.copy(alpha = 0.6f)
+private val SecondaryTextColor @Composable get() = MaterialTheme.colors.onSurface.copy(alpha = 0.46f)
+private val SecondaryFontSize = 13.sp
+private val PrimaryFontSize = 17.sp
 
 @Composable
-fun DomainItem(
-    uiState: DomainUiState,
-) = with(uiState) {
+fun DomainItem(uiState: DomainUiState) = with(uiState) {
     Column {
         Row(
+            verticalAlignment = Alignment.CenterVertically,
             modifier = Modifier
                 .clickable { onClick.invoke() }
-                .padding(horizontal = 24.dp, vertical = 16.dp)
+                .padding(vertical = Margin.ExtraLarge.value)
+                .padding(end = Margin.ExtraLarge.value)
         ) {
-            Text(domainName)
-            Spacer(modifier = Modifier.weight(1f))
-            Text(
-                text = cost,
-                color = CostColorDefault,
-            )
+            Box(
+                contentAlignment = Alignment.Center,
+                modifier = Modifier.width(Margin.ExtraMediumLarge.value)
+            ) {
+                variant?.dotColor?.let {
+                    SolidCircle(size = 8.dp, colorResource(it))
```

Comment:
My understanding is that the reasoning for inlining dimension values is [similar with the previous PR](https://github.com/wordpress-mobile/WordPress-Android/pull/17998#discussion_r1114999461). Correct?

Reply:
Correct 🙇🏻 , we're (Ajesh and I) probably going do start a discussion (internally) on this very soon 👍🏻